components/layout/Settings.js

```diff
@@ -1,14 +1,14 @@
-import "./Settings.scss";
+import styles from "../../styles/layout/Settings.module.scss"
```

Comment:
Use lowercase for the first letter of file name

Reply:
I adjust all above styles with variables. Please check.

Reply:
OK